### PR TITLE
Set up rubocop for travis

### PR DESCRIPTION
**Pull request by ondenman**
This PR is in preparation of refactoring the scraper to use `scraped` (WIP:#3 )

This enables Travis to run rubocop checks against PRs.

Note: I have not enabled Travis as the current code will not pass.

### CODE REVIEW

File: .rubocop-https---raw-githubusercontent-com-everypolitician-everypolitician-data-master--rubocop-base-yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# A base Rubocop configuration that other repos can inherit from
+# http://rubocop.readthedocs.io/en/latest/configuration/#inheriting-configuration-from-a-remote-url
```
Comment (tmtmtmtm) [inline .rubocop-https---raw-githubusercontent-com-everypolitician-everypolitician-data-master--rubocop-base-yml]: Looks like you checked this in by mistake…

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]
```
> bundle exec rake   
rake aborted!
Don't know how to build task 'rubocop' (see --tasks)
```

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]
> Note: I have not enabled Travis as the current code will not pass.

We shouldn't be merging code that fails `bundle exec rake`. This isn't about whether it fails *on travis* — travis is a way automating that check.

**Comment by ondenman** [comment]
> We shouldn't be merging code that fails `bundle exec rake`. This isn't about whether it fails *on travis* — travis is a way automating that check. 

@tmtmtmtm Should I cherry pick these across to #3 then?

**Comment by tmtmtmtm** [comment]
@ondenman I've run my semi-automatic moderniser as https://github.com/everypolitician-scrapers/hong_kong_legislative_council_members/pull/5

**Pull request closed without merging**